mathics/builtin/numbers/calculus.py

```diff
@@ -664,8 +668,8 @@ def eval(self, f, x, x0, evaluation: Evaluation, options: dict):
 
         # Determine the "jacobian"s
         if (
-            method in ("Newton", "Automatic")
-            and options["System`Jacobian"] is SymbolAutomatic
+            method in ("Newton", "Automatic") and
```

Comment:
This looks like a fight between versions of black / autopep8.

Reply:
In Github Actions, we are using the following versions:
 'click==8.0.4' 'black==22.3.0' 'isort==5.10.1'